lib/Calculation.ts

```diff
@@ -0,0 +1,71 @@
+import type { Position } from "./configMeta.ts";
+import config from "../config.ts";
+import { log10 } from "mathjs";
```

Comment:
You are importing a library to do this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/log10